paymentsheet/src/main/java/com/stripe/android/link/ui/inline/InlineSignupViewModel.kt

```diff
@@ -63,18 +63,24 @@ internal class InlineSignupViewModel(
 
     private val hasInitialUserInput = initialUserInput != null
 
-    private val initialViewState = InlineSignupViewState.create(signupMode, config, hasInitialUserInput)
+    private val initialEmail = initialUserInput?.email()
+    private val initialPhone = initialUserInput?.phone()
+    private val initialName = initialUserInput?.name()
+    private val initialCountry = initialUserInput?.country()
+
+    private val initialViewState = InlineSignupViewState.create(
+        signupMode = signupMode,
+        config = config,
+        initialEmail = initialEmail,
+        initialPhone = initialPhone,
+        isExpanded = hasInitialUserInput,
+    )
```

Comment:
should we just store `initialUserInput` entirely on the state instead?

Reply:
Are you ok with this happening as a follow-up?